components/script/dom/webidls/VRDisplayEvent.webidl

```diff
@@ -0,0 +1,22 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// https://w3c.github.io/webvr/#interface-vrdisplayevent
+
+enum VRDisplayEventReason {
+  "navigation",
+  "mounted",
+  "unmounted",
```

Comment:
can you also support `requested` (see [this spec commit](https://github.com/w3c/webvr/commit/e4c839)?